rpcs3/Emu/RSX/GL/GLOverlays.cpp

```diff
@@ -603,11 +603,17 @@ namespace gl
 			"		ocol = color;\n"
 			"}\n";
 
-		input_filter = GL_LINEAR;
+		m_input_filter = gl::filter::linear;
 	}
 
-	void video_out_calibration_pass::run(gl::command_context& cmd, const areau& viewport, const rsx::simple_array<GLuint>& source, f32 gamma, bool limited_rgb, bool _3d)
+	void video_out_calibration_pass::run(gl::command_context& cmd, const areau& viewport, const rsx::simple_array<GLuint>& source, f32 gamma, bool limited_rgb, bool _3d, gl::filter input_filter)
 	{
+		if (m_input_filter != input_filter)
+		{
+			m_input_filter = input_filter;
```

Comment:
I think the member should stay a GLenum and then you should use a switch at the beginning of the function to set the enum properly.

Then you also won't have do those static casts everywhere, which fit by coincidence at best.

Reply:
I felt the enum was more explicit here and based the static casting off of https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/RSX/GL/glutils/fbo.cpp#L29

Reply:
For the record, a lot of the gl:: namespace enums are rather pointless and I'm not the one who came up with them. They only make sense for the purposes of type checking, i.e making sure the correct type of enum is passed to a function. But in this case its being used for storage, there is really no reason to keep it an enum.
The only benefit I can think of is that msvc is really bad at inferring GL_XXX enums during debug sessions.

Reply:
TLDR, it's fine.

Reply:
Ah. for some reason I though the parameter was a config param when I read this at 1am xD